README.md

```diff
@@ -105,7 +105,7 @@ Go build tags can tweak certain functionality at compile-time. These are for adv
 have compatibility guarantees across minor versions - use with care.
 
 - coraza.disabled_operators.* - excludes the specified operator from compilation. Particularly useful if overriding
-the operator with `operators.Register` to reduce binary size / startup overhead.
+the operator with `plugins.RegisterOperator` to reduce binary size / startup overhead.
```

Comment:
With https://github.com/corazawaf/coraza/pull/734 operators have been moved to an internal package, from an user standpoint it is better to point out directly `plugins.RegisterOperator`, isn't it?